kubernetes/kube-state-metrics-deployment.yaml

```diff
@@ -28,6 +28,6 @@ spec:
             memory: 30Mi
             cpu: 100m
           limits:
-            memory: 50Mi
+            memory: 2Gi
```

Comment:
I'm using https://github.com/wercker/cronetes because GKE do not supports cron jobs.
And, for some reason, when the job fail the POD aren't been removed and I only detected the bug because this project. In my cluster there was more than 6k dead POD. And, probably because of it the `kube-state-metrics` are using a lot of memory. 

I fixed all wrong jobs of my cluster and I'll test kube-state-metrics again.